lithops/localhost/localhost.py

```diff
@@ -41,15 +41,16 @@ class LocalhostHandler:
     """
 
     def __init__(self, localhost_config):
-        logger.debug('Creating Localhost compute client')
+        logger.info('Creating Localhost compute client')
         self.config = localhost_config
         self.runtime = self.config['runtime']
+        self.local_runtime_load = self.config['local_runtime_load']
```

Comment:
@gilv I think LocalExecutor will fail due this. Better change it to: `self.local_runtime_load = self.config.get('local_runtime_load', False)`